andela/rogue-backend
# #167727614 Configures ESLint for the project

**What does this PR do?**
Configures Eslint for the project
**Description of Task Taken to Complete?**
Installed these development dependencies: **"eslint": "^5.16.0"**,
**"eslint-config-airbnb-base": "^14.0.0"**,
**"eslint-config-prettier": "^6.0.0"**,
**"eslint-plugin-import": "^2.18.2"**
Created a .eslintrc.json file at the project's root directory and set up rules as described on the PT board.
**How should this be manually Tested?**
Clone the repo, run npm install, close your editor and open again.
Check if the Eslint server is running and notice red wiggles for violated rules.

**Link to pivotal tracker story:**
https://www.pivotaltracker.com/n/projects/2381976/stories/167727614
